g2p/apply.py

```diff
@@ -70,49 +67,41 @@ def __init__(
         self.out_g2p_lexicon = self.output_path("g2p.lexicon")
         self.out_g2p_untranslated = self.output_path("g2p.untranslated")
 
-        if self.concurrent > 1:
-            self.out_g2p_lexicon_parts = [
-                self.output_path(f"g2p.lexicon.{i}")
-                for i in range(1, self.concurrent + 1)
-            ]
-            self.out_g2p_untranslated_parts = [
-                self.output_path(f"g2p.untranslated.{i}")
-                for i in range(1, self.concurrent + 1)
-            ]
-            num_digits = len(str(self.concurrent))
-            self.word_list_parts = [
-                f"words.{i:0{num_digits}d}" for i in range(1, self.concurrent + 1)
-            ]
+        self.out_g2p_lexicon_parts = [
```

Comment:
Please use the `out_` prefix only for actual outputs, here those are just intermediate variables. I would also recommend to just declare them private, so `self._g2p_lexicon_parts`.

Reply:
as the names do not change, you might even not have these as members of the job at all, but create them as local variable in the respective task. Or refactor the loops to loop over integers and then open the file `f"g2p.lexicon.{i}"` instead of looping over the filenames if that is easier.

Also, now that these guys are not outputs any longer, they should - if you decide to keep them - be move above the definition of the outputs.

Reply:
Done.